release-matrix/release-matrix.json

```diff
@@ -1,10 +1,18 @@
 [
   {
-    "mkeReleaseVersion": "v4.0.1-alpha.1",
+    "mkeReleaseVersion": "v4.0.1-alpha.2",
     "blueprintOperatorVersion": "v1.0.18",
-    "mkeOperatorVersion": "v1.2.0",
+    "mkeOperatorVersion": "v1.2.1",
     "k0sVersion": "v1.31.1+k0s.1",
     "isLatest": true,
+    "minimumMkectlVersion": "v4.0.1-alpha.2"
+  },
+  {
+    "mkeReleaseVersion": "v4.0.1-alpha.1",
+    "blueprintOperatorVersion": "v1.0.17",
+    "mkeOperatorVersion": "v1.2.0",
```

Comment:
@RaunakShah 
how is this supposed to work if all versions always pull the latest MOP? Should I also set it to v1.2.1 even though when the first alpha was released, the latest MOP version was v1.2.0?

Reply:
No it should remain as `v1.2.0`. All versions pull the latest at the time of their release. 
So if a user wants to deploy `v4.0.1-alpha.1`, they should always get MOP `v1.2.0`.